rq/job.py

```diff
@@ -393,6 +398,35 @@ def dependents_key(self):
         """The Redis key that is used to store job dependents hash under."""
         return self.dependents_key_for(self.id)
 
+    @property
+    def dependencies_key(self):
+        return '{0}{1}:dependencies'.format(self.redis_job_namespace_prefix, self.id)
```

Comment:
The key should be `{0}:{1}:dependencies`

Reply:
In  7c690087a9f4d287d400e8b489eb851e94fedfc0:

https://github.com/rq/rq/blob/7c690087a9f4d287d400e8b489eb851e94fedfc0/rq/job.py#L403